src/routes/external-extra/components/ExternalExtra.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import * as React from 'react'
+import { WebView } from 'react-native-webview'
+import { Text } from 'react-native'
+import { stringify } from 'query-string'
+
+export type PropsType = {|
+  url: ?string,
+  postData: ?Map<string, string>
+|}
+
+class ExternalExtra extends React.Component<PropsType> {
+  render () {
+    const {url, postData} = this.props
+    // $FlowFixMe Object.fromEntries is available in Flow version >= 0.99.x: https://github.com/facebook/flow/commit/f92231cf15022c729226b4d440435b3366f9346c
+    const body = !postData ? '' : stringify(Object.fromEntries(postData))
```

Comment:
I think we should avoid adding FlowFixMes if there is an easy workaround like a `reduce`. I mean yes it is not pretty, but we will never have time to fix cosmetics like this ^^

Reply:
Well, this FlowFixMe is exceptionally in the manner, that I already know that it's going to be fixed with a newer flow version, so I'd reject your comment from that point of view :) Also lodashs reduce does not work with a map or es6 iterators out of the box... But in fact, it might be possible using the array destructor and lodash's `toPairs`... I'll give it a try...

Reply:
Did work 👍